WordPressAuthenticatorTests/Credentials/CredentialsTests.swift

```diff
@@ -33,32 +33,37 @@ class CredentialsTests: XCTestCase {
-                                          isJetpackLogin: false,
-                                          multifactor: false,
-                                          siteURL: siteURL)
-        
-        XCTAssertTrue(lhs == rhs)
-    }
-    
-    func testWordPressComCredentialsNotEquatable() {
-        let lhs = WordPressComCredentials(authToken: token,
-                                          isJetpackLogin: false,
-                                          multifactor: false,
-                                          siteURL: siteURL)
-        
-        let rhs = WordPressComCredentials(authToken: token,
-                                          isJetpackLogin: false,
-                                          multifactor: false,
-                                          siteURL: "")
-        
-        XCTAssertFalse(lhs == rhs)
+    func testWordPressComCredentialsEquatableReturnsCorrectValue() {
+        let credential = WordPressComCredentials(authToken: token,
+                                                 isJetpackLogin: false,
+                                                 multifactor: false,
+                                                 siteURL: siteURL)
+        let match = WordPressComCredentials(authToken: token,
+                                                 isJetpackLogin: false,
+                                                 multifactor: false,
+                                                 siteURL: siteURL)
+        let differentJetpack = WordPressComCredentials(authToken: token,
+                                                 isJetpackLogin: true,
+                                                 multifactor: false,
+                                                 siteURL: siteURL)
+        let differentMultifactor = WordPressComCredentials(authToken: token,
+                                                 isJetpackLogin: false,
+                                                 multifactor: true,
+                                                 siteURL: siteURL)
+        let differentSiteURL = WordPressComCredentials(authToken: token,
+                                                       isJetpackLogin: false,
+                                                       multifactor: false,
+                                                       siteURL: "")
+        let differentAuthToken = WordPressComCredentials(authToken: "ARSTDBVCXZ(*&^%$",
+                                                         isJetpackLogin: false,
+                                                         multifactor: false,
+                                                         siteURL: siteURL)
+
+        XCTAssertTrue(credential == match)
+        XCTAssertTrue(credential == differentJetpack)
+        XCTAssertTrue(credential == differentMultifactor)
+        XCTAssertFalse(credential == differentSiteURL)
+        XCTAssertFalse(credential == differentAuthToken)
```

Comment:
I'd recommend using `XCTAssertEqual` and `XCTAssertNotEqual`.

The kind of assertions you use makes a difference in how understandable the test failures will be.
Consider these two assertions verifying the same condition:

```swift
let sections: [Section] = getSections()

XCTAssertTrue(sections.count == 1)
XCTAssertEqual(sections.count, 1)
```

If the `sections` array has more than one element, the first will fail with:

```
XCTAssertTrue failed
```


while the second will fail with:

```
XCTAssertEqual failed: ("1") is not equal to ("2")
```

Which one do you think better represent that the array count was not the expected one?

Reply:
I agree I think it makes more sense to use the equal/not equal for these tests.  I updated the tests